ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -8,16 +8,7 @@ import { tracked } from '@glimmer/tracking';
 import { inject as service } from '@ember/service';
 import { action } from '@ember/object';
 import { A } from '@ember/array';
-
-class Tag {
-  @tracked key;
-  @tracked value;
```

Comment:
The new tag class doesn't use tracked fields anymore which changed the functionality in the create worker form, the tags don't update immediately anymore

Reply:
Oops. Thanks for catching that. Tested and working.